django/contrib/sessions/backends/db.py

```diff
@@ -14,14 +15,25 @@ class SessionStore(SessionBase):
     def __init__(self, session_key=None):
         super(SessionStore, self).__init__(session_key)
 
+    @classmethod
+    def get_session_class(cls):
```

Comment:
In this context, I think "session_model" is more meaningful. I think it's not intuitive what "session_class" means. If we follow the pattern elsewhere, I guess "get_model_class" could be the way to go.